docs/build/tooling/foundry/getting-started.md

```diff
@@ -81,26 +81,26 @@ libs = ['lib']
 src = 'src'
 libs = ['lib']
 fallback_oz = true
-is_system = true
+is_system = false
 mode = "3"
 ```
 
 ## Basic Usage
 
 ### Running Tests
 
-Use `zkforge test` to run tests written for your smart contracts. For an overview of how to write tests using `Foundry-zksync` please refer to Foundry testing [here](../../test-and-debug/foundry.md).
+Use `forge test --zksync` to run tests written for your smart contracts. For an overview of how to write tests using `Foundry-zksync` please refer to Foundry testing [here](../../test-and-debug/foundry.md).
```

Comment:
I know this has been here, but should Foundry-zksync be lowercased to `foundry-zksync`?

Reply:
Updated to keep consistent with lowercase usage.